juno/assetlist.json

```diff
@@ -2166,6 +2166,51 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/wind.png"
         }
       ]
+    },
+	{
+      "description": "Airdrop For All [AFA - New Name on Cosmos Ecosystem, A4A - Old Name on TurtleNetwork] is a token from turtleNetwork towards cosmos ecosystem.",
+      "denom_units": [
+        {
+          "denom": "ibc/juno1spjes0smg5yp40dl7gqyw0h8rn03tnmve06dd2m5acwgh6tlx86swha3xg",
+          "exponent": 0,
+          "aliases": [
+            "afa"
+          ]
+        },
+        {
+          "denom": "afa",
```

Comment:
The denom_units still don't make sense. There can't be two different units called 'afa'.
Is this asset divisible? If indivisible, remove the second denom_unit, and change the display value to match the base denom ('ibc/juno...').